### PR TITLE
Cgi timeout

### PR DESCRIPTION
**Pull request by rosamakinen**

### CODE REVIEW

File: include/CgiHandler.hpp
Comment (forbidden-arts) [inline include/CgiHandler.hpp:1]: most/or all of these standard includes exist in "WebServer.hpp" can you #include that instead?

File: src/Util.cpp
Comment (forbidden-arts) [inline src/Util.cpp:1]: double include. super quick fix

**Review by forbidden-arts — APPROVED** [review]
tiny changes from me. otherwise looks solid.

i would like to discuss moving timeout() to the utils class, since we'll have to have a similar timeout thing for requests (maybe even using the same func with different args?)

File: include/CgiHandler.hpp
Comment (merituulie) [reply include/CgiHandler.hpp:1]: you should not. What if we remove the webserv include at some point?

Comment (forbidden-arts) [reply include/CgiHandler.hpp:1]: Fair point, I was imagining webserv.hpp as a spot where we stick all of our defines and the <standard> includes. I suppose we don't get penalized for the norm anymore so extra includes are no biggie

File: src/Util.cpp
Comment (rosamakinen) [reply src/Util.cpp:1]: good catch, I'll remove it from the next PR!

Comment (rosamakinen) [reply src/Util.cpp:1]: and I can check with meri if it feels like it's easier to have a common utils timer for of us. I think she already has an implementation ready for her own timing needs, but we can do the joined one as we polish off our webserv, if it feels like a good optimization.

**Review by merituulie — APPROVED** [review]

**Pull request merged**